editor/src/components/canvas/controls/yoga-control.tsx

```diff
@@ -111,21 +128,45 @@ export class YogaControls extends React.Component<YogaControlsProps> {
         this.props.componentMetadata,
       )
     })
-    const unknownElementsSelected = MetadataUtils.anyUnknownOrGeneratedElements(
```

Comment:
I see this function is used from two places, wouldn't it be better to keep `MetadataUtils.anyUnknownOrGeneratedElements` alive and update it with the below code?

Reply:
Yeah, I was a bit dubious of pulling the project contents types and the like into `MetadataUtils`, but there appear to be no mild explosions at doing that.

Reply:
Fixed in 70316f2.